src/App.tsx

```diff
@@ -82,6 +82,8 @@ const App = () => {
     'expo-permissions is now deprecated',
   ])
 
+  LogBox.ignoreAllLogs()
```

Comment:
We should remove this. Are there lots of errors now regarding non-serializable objects?

Reply:
Whoops forgot to push my commit to remove this. Was just using it for testing with offline network